.github/workflows/reusable-build-and-push.yml

```diff
@@ -43,6 +43,11 @@ jobs:
             ${{ ((inputs.tag != '') && format('{0}/ecamp3-frontend:{1}', vars.DOCKER_HUB_USERNAME, inputs.tag) || '') }}
             ${{ vars.DOCKER_HUB_USERNAME }}/ecamp3-frontend:${{ inputs.sha }}
           context: .
+          build-args: |
+            SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+            SENTRY_ORG: ${{ vars.SENTRY_ORG }$
```

Comment:
Isn't that a typo and should end with `}}` instead of `}$`? And same for the change below in print

Reply:
you are right, done